builder/dockerfile/internals.go

```diff
@@ -298,16 +297,30 @@ func (b *Builder) download(srcURL string) (fi builder.FileInfo, err error) {
 	return &builder.HashedFileInfo{FileInfo: builder.PathFileInfo{FileInfo: tmpFileSt, FilePath: tmpFileName}, FileHash: hash}, nil
 }
 
+var windowsBlacklist = append([]string{}, "c:\\", "c:\\windows")
```

Comment:
I'll go with the map solution to avoid looping (might become annoying if we put many system paths in the blacklist)